main.py

```
@@ -596,6 +602,33 @@ def _prepare_whitelist(self) -> set:
+                    item_type = getattr(item, 'type', '') if hasattr(item, 'type') else ''
+                    if item_type in ('image', 'image_url'):
+                        has_sticker = True
+                        break
+
+            if msg_text or has_sticker:
+                self.behavior_profile.feed(
+                    msg_text, user_id=user_id, scope=scope,
+                    has_sticker=has_sticker,
+                )
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)